src/seer.app.src

```diff
@@ -8,7 +8,7 @@
         {mod, {seer_app, []}},
         {applications, [kernel, stdlib]},
         {env, []},
-        {modules, []},
+        {modules, [seer, seer_server, seer_sup]},
```

Comment:
`seer_utils` should go here too? I've never been in the habit of maintaining this field.